driver.go

```diff
@@ -131,6 +131,144 @@ func (s *Session) processTriple(tx neo4j.Transaction, edge graph.Edge, nodeIds m
 	return nil
 }
 
+func (s *Session) LoadEntityNodes(tx neo4j.Transaction, grph graph.Graph, rootIds []int64, config Config) (graph.Graph, error) {
```

Comment:
Write a cobra.Command that calls this that looks like:

lsaneo load 1 2 3 4 5...

Output will be a graph